data/json/npcs/TALK_ALLY_TUTORIAL.json

```diff
@@ -0,0 +1,198 @@
+    "responses": [
+      { "text": "Thanks for the explanation.  I wanted to know more about something else.", "topic": "TALK_ALLY_TUTORIAL" },
+      { "text": "Thanks.  I have some things for you to do.", "topic": "TALK_FRIEND" },
+      { "text": "Skip it, let's get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_ALLY_TUTORIAL_MULE",
+    "type": "talk_topic",
+    "dynamic_line": "You give me something to carry, I'll carry it.  But I only have so many pockets and I'm only so strong, so I'll drop stuff that's too big to carry. \nI'll also wear stuff - I'll decide what I want to wear, but you can take stuff from me.  If I'm wearing something I shouldn't, you can bump into me and sort my armor to get me to take if off. \nI don't like wearing a lot of gear, so if you give me a lot of bulky stuff and things that don't fit, I'm going to take them off and probably drop them. \nAlso, since we're friends, I'll give you anything I'm carrying, no questions asked. \nOh, yeah, if I'm hungry or thirsty and I'm carrying food, I'll eat it.  So you might want to watch what you give me, y'know?",
+    "responses": [
+      { "text": "Thanks for the explanation.  I wanted to know more about something else.", "topic": "TALK_ALLY_TUTORIAL" },
+      { "text": "Thanks.  I have some things for you to do.", "topic": "TALK_FRIEND" },
+      { "text": "Skip it, let's get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_ALLY_TUTORIAL_MEDIC",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "and": [
+        {
+          "npc_has_trait": "PROF_MED",
+          "yes": "Hey, I'm a doctor!  I know how to treat trauma.  You give me some bandages or a bottle of disinfectant, I'm get you fixed when I see you hurting.",
+          "no": "Yeah, I can perform first aid.  You give me some bandages or a bottle of disinfectant, I'll treat your wounds as best I can."
+        },
+        "  But remember, I like you, but I like me more - so I'm going to use those items on myself first, then you, then any of your other friends."
+      ]
+    },
+    "responses": [
+      { "text": "Thanks for the explanation.  I wanted to know more about something else.", "topic": "TALK_ALLY_TUTORIAL" },
+      { "text": "Thanks.  I have some things for you to do.", "topic": "TALK_FRIEND" },
+      { "text": "Skip it, let's get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_ALLY_TUTORIAL_MOVEMENT",
+    "type": "talk_topic",
+    "dynamic_line": "If you call me by radio, you can tell me to come to you. If you've got some basecamps set up, you can tell me to go to one of them.  I'll start walking, and when I get to where I'm going, I'll guard it. \nNow depending on where I am and where I'm going, it may take me a while to get there.  I'll dodge any dangers on the way, don't worry about that, but if you're way the heck away from me and tell me to come, it may be a while until I get to you.",
```

Comment:
```suggestion
    "dynamic_line": "If you call me by radio, you can tell me to come to you.  If you've got some basecamps set up, you can tell me to go to one of them.  I'll start walking, and when I get to where I'm going, I'll guard it. \nNow depending on where I am and where I'm going, it may take me a while to get there.  I'll dodge any dangers on the way, don't worry about that, but if you're way the heck away from me and tell me to come, it may be a while until I get to you.",
```